packages/client/src/components/StyledErrorBoundary.tsx

```diff
@@ -47,6 +47,10 @@ const StyledErrorBoundaryComponent = ({ intl, children }: IFullProps) => {
   const [authError, setAuthError] = useState(false)
 
   const onError = (error: Error) => {
+    if (import.meta.env.MODE === 'test') {
```

Comment:
**Improvement:** related to https://github.com/opencrvs/opencrvs-core/pull/7301/files#r1667055909